README.md

```diff
@@ -1,5 +1,5 @@
 
-# data.table <a href="http://r-datatable.com"><img src="logo.png" align="right" height="140" /></a>
+# data.table <a href="https://rdatatable.gitlab.io/data.table"><img src="https://raw.githubusercontent.com/Rdatatable/data.table/master/.graphics/logo.png" align="right" height="140" /></a>
```

Comment:
@jangorecki Shall I redirect r-datatable.com to rdatatable.gitlab.io/data.table now?  The idea was that r-datatable.com was the canonical URL to publicize and link, then I can just change where it forwards to, like now.  If so, the logo can be left linking to r-datatable.com ?

Reply:
yes, I think we can redirect http://r-datatable.com to rdatatable.gitlab.io/data.table

Reply:
Done.